<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainSettingClass</class>
 <widget class="QDialog" name="MainSettingClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>370</width>
    <height>220</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>主账号设置</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>30</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>账 号</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="inverstID">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>20</y>
     <width>110</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>20</y>
     <width>30</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>密 码</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="password">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>20</y>
     <width>110</width>
     <height>20</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>65</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>经纪商编号</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>65</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>服务器名称</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>65</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>服务器地址</string>
   </property>
  </widget>
  <widget class="QComboBox" name="brokerName">
   <property name="geometry">
    <rect>
     <x>85</x>
     <y>60</y>
     <width>180</width>
     <height>20</height>
    </rect>
   </property>
   <property name="editable">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="saveButton">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>180</y>
     <width>81</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>保  存</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>180</y>
     <width>81</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>取  消</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="brokerID">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>85</x>
     <y>100</y>
     <width>180</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="frontAddr">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>85</x>
     <y>140</y>
     <width>265</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>saveButton</sender>
   <signal>clicked()</signal>
   <receiver>MainSettingClass</receiver>
   <slot>saveBtnClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>240</x>
     <y>202</y>
    </hint>
    <hint type="destinationlabel">
     <x>96</x>
     <y>209</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>MainSettingClass</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>349</x>
     <y>202</y>
    </hint>
    <hint type="destinationlabel">
     <x>179</x>
     <y>209</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>brokerName</sender>
   <signal>currentTextChanged(QString)</signal>
   <receiver>MainSettingClass</receiver>
   <slot>BrokerNameselected(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>135</x>
     <y>67</y>
    </hint>
    <hint type="destinationlabel">
     <x>305</x>
     <y>72</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>saveBtnClicked()</slot>
  <slot>BrokerNameselected(QString)</slot>
 </slots>
</ui>
